src/stat.cc

```diff
@@ -327,8 +333,8 @@ statObjects(void *data)
     StoreEntry *e;
 
     if (state->theSearch->isDone()) {
-        if (UsingSmp())
-            storeAppendPrintf(state->sentry, "} by kid%d\n\n", KidIdentifier);
+        if (state->onCompleteHandler)
+            state->onCompleteHandler(state->sentry);
```

Comment:
I see not good reason to add a third parameter to statObjectsStart() when all statObjectsStart() get the same third parameter value and all StatObjectsState instances get that same value. AFAICT, this `state->onCompleteHandler` is always true. AFAICT, all these changes should be removed and this code will look like this:

```suggestion
        Mgr::closeKidSection(state->sentry, false); // we have not converted these reports to YAML yet
```

Reply:
That call to closeKidSection is currently done manually in statObjects,
at the test for state->theSearch->isDone()).
We need to close the framing at that point for non-atomic non-aggregatable actions,
of which at this time there is exactly one but we might have more in the future, this is an
attempt to have a generic(er) solution. It was sparked by me failing to understand why
there was a section being closed there with no corresponding opening (it is in FunAction)